Cargo.toml

```diff
@@ -15,4 +15,4 @@ categories = ["development-tools::build-utils"]
 edition = "2021"
 
 [dependencies]
-cc = "1.0.83"
+cc = "1.1.0"
```

Comment:
I didn't update this to anything specific, just a version that I know has the fix. I guess if you wanted, I could do a bit more research and find the precise version that fixes it.